src/main/resources/web_en.properties

```diff
@@ -21,9 +21,9 @@ MailService.invalidReplyTo = Invalid reply-to address: ${address}
 MailService.invalidSender = Invalid sender address: ${address}
 Model.next = More
 Page.noResults = no results
-Page.range = ${first} – ${last}[ of ${total}]
-Pagination.next = Next
-Pagination.previous = Previous
+Page.range = ${first} - ${last} [from ${total}]
```

Comment:
Do we want to rewrite those? 1 of 10 feel more fitting than 1 from 10, but can live with that. Same for the next and back below, but affects all other languages.

Reply:
Ne, das sind alles Verschlimmbesserungen. Bitte die drei so belassen.

Reply:
Ja, aber die Bedeutung für die Andere Sprachen, wo noch keine Übersetzung gibt, ist dann falsch.
Kann man die Lokalise andere _meaning_ auswählen?